en/migrations.rst

```diff
@@ -204,14 +204,17 @@ Columns definition
 When using columns in the command line, it may be handy to remember that they
 follow the following pattern::
 
-    fieldName:fieldType[length]:indexType:indexName
+    fieldName:fieldType?[length]:indexType:indexName
 
 For instance, the following are all valid ways of specifying an email field:
 
+* ``description:string?``
```

Comment:
Shall I change the header to "For instance, the following are all valid ways of specifying a field:" ? That way, we can later provide examples with different data types too. BTW, the field type is optional and defaults to string (except for the columns id, created, modified etc.).